SR2024/2023-09-23-place-confirmation.md

```diff
@@ -0,0 +1,51 @@
+---
+to: Student Robotics 2024
+subject: Confirming your place at Student Robotics 2024
+---
+
+You're in! We're happy to confirm you've got a place at Student Robotics 2024!
+
+As the new school year starts, so does Student Robotics. The year starts with a
+"Kickstart" event which gives competitors and team supervisors an introduction
+to this year's competition and overview of how to build and engineer their
+robot.
+
+At various points in the year we will have a number of "Tech Days" at locations
+around the UK. These provide an opportunity for your team to get hands-on help
+from our volunteer mentors, make progress on their robot and meet other teams.
+
+Throughout the year your team will be able to get support for our mentors via
+[Discord](https://studentrobotics.org/docs/tutorials/discord). In addition to
+shared channels where any team can ask questions, each team will have their own
+private text chat channel and team supervisors can request voice & video
+channels too. These offer a space for teams to discuss their robot and get
+direct support from our mentors at any time. We will send out invites to Discord
+separately.
```

Comment:
Suggestion: We should be explicit that these will arrive in time for kickstart.

```suggestion
direct support from our mentors at any time. We will send out invites to Discord
separately before kickstart.
```

Reply:
9580a05f4ba69217f3cb2c16a912f498a2f0253e reiterates that this information is coming separately. I hope it's clear from the context that that's before kickstart (especially given that we talk about them using Discord during the event).